<?xml version="1.0"?>
<launch>
    <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

    <node pkg="nodelet" type="nodelet" name="LandoltNodelet" args="load capra/LandoltNodelet standalone_nodelet" output="screen">
        <param name="subscriber_image" type="str" value="/capra/camera_3d/rgb/image_raw" />
    </node>
</launch>